src/ocgis/conv/nc.py

```diff
@@ -169,14 +169,13 @@ def _preformatting_(self, i, coll):
                                )
             )
 
-            if 'ocgis_spatial_mask' in field:
-                # Remove the spatial_mask and replace by new one.
-                field.remove_variable('ocgis_spatial_mask')
+            if VariableName.SPATIAL_MASK in field:
+                # Remove the spatial_mask.
+                field.remove_variable(VariableName.SPATIAL_MASK)
+                field.dimension_map.set_spatial_mask(None)
```

Comment:
@huard: The spatial mask variable retrieval is strict when it is listed in the dimension map. This should be relaxed (going to think about that), but in the meantime this is fully correct. If these changes work for you, I'll merge!